.github/workflows/tiledb-py.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         shell: bash -l {0}
         run: bash ci/scripts/rerender-feedstock.sh tiledb-py-feedstock
       - name: Push update to GitHub
-        if: ${{ github.ref == 'refs/heads/main' && github.repository == 'TileDB-Inc/conda-forge-nightly-controller' && github.event_name != 'pull_request' }}
+        if: github.ref == 'refs/heads/main' && github.repository_owner == 'TileDB-Inc' && github.event_name != 'pull_request'
```

Comment:
Also note that I confirmed expression syntax `${{ }}` is optional for `if:` since it automatically [assumes it is an expression](https://docs.github.com/en/actions/learn-github-actions/expressions#operators). Removing it makes it more readable